db/migrations/000002_create-events-table.up.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE IF NOT EXISTS organizations (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(100) NOT NULL DEFAULT 'self'
+);
+
+CREATE TABLE IF NOT EXISTS images (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    filename VARCHAR(255),
+    url VARCHAR(255),
+    alt VARCHAR(150),
+    uploaded_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
```

Comment:
I'm wondering if we should name this `created_at`. We should stick to using the same name in all the tables so that it's consistent. wdyt?

Reply:
You're right, thanks for catching this! I will rename this field to `created_at`.